scripts/take_screenshot.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# This script takes a screenshot of the connected device using adb.
+# The screenshot is saved in the adb-screenshots directory with the current date and time as the file name.
+# See https://android.googlesource.com/platform/frameworks/base/+/master/packages/SystemUI/docs/demo_mode.md
+
+TARGET_DIR="./adb-screenshots/"
+FILE_NAME="$(date +"%Y-%m-%d_%H-%M-%S").png"
+
+start_demo_mode() {
+    adb shell settings put global sysui_demo_allowed 1
+    adb shell am broadcast -a com.android.systemui.demo -e command enter
+    adb shell am broadcast -a com.android.systemui.demo -e command clock -e hhmm 1200
```

Comment:
Do we want to add an Easter egg here? 00:58 was the time of the initial commit to this repo for example. 20:08 would be the year. 10:28 for the date of the first release, October 28th. I like the last one most personally :)

Reply:
I think first commit^^